DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Helpers/RevitConversionContextStack.cs

```diff
@@ -0,0 +1,22 @@
+using Autodesk.Revit.DB;
+using Speckle.Converters.Common;
+
+namespace Speckle.Converters.RevitShared.Helpers;
+
+[System.Diagnostics.CodeAnalysis.SuppressMessage(
+  "Naming",
+  "CA1711:Identifiers should not have incorrect suffix",
+  Justification = "See base class justification"
+)]
+public class RevitConversionContextStack : ConversionContextStack<UI.UIDocument, ForgeTypeId>
+{
+  public double TOLERANCE { get; } = 0.0164042; // 5mm in ft
```

Comment:
I feel this should be a constant unless it can change in any way?

Reply:
you are correct